entrypoint/99-crond.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -u
```

Comment:
```suggestion
set -eu
```

Reply:
I believe the -e flag doesn't do much here as well having only one command inside the script, but added for consistency.